src/main/java/com/GotYourBac/gotYourBac/models/Drink.java

```diff
@@ -1,27 +1,32 @@
 package com.GotYourBac.gotYourBac.models;
 
-import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
-import javax.persistence.Id;
+import javax.persistence.*;
 
 @Entity
 public class Drink {
 
+    //TODO: probably need some getters and setters depending on the app, but lets safely add each one we need throughout the process.
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     long id;
 
+    @ManyToOne
```

Comment:
if users do not create drinks that they own, there may not need to be a pointer on each drink back to the appUser

Reply:
whereas if drinks are really are a reference to physically consumed beverages, you have the relationship correct

Reply:
Users will input their consumed drinks so that the BAC calculator will specified for the user and their input of drinks